docs/best_practices.rst

```diff
@@ -0,0 +1,48 @@
+#######################################
+edx-platform Development Best Practices
+#######################################
+
+There are many general best practices documented for `Open edX Development in
+Confluence`_. The following best-practices are specific to edx-platform.
+
+Course Access in LMS
+********************
+
+When coding in the LMS, there are several technologies that are preferred for
```

Comment:
I found it confusing to talk about the benefits of technologies that haven't yet been enumerated. Maybe have a simple enumeration of the technologies first, along these lines:

    the following technologies can be used to access course-related data:

      * *Course Overviews* - brief description...
      * *Course Blocks* - brief description...
      * *Modulestore* - brief description...